scripts/measures/archived/scripts/get-stratification.js

```diff
@@ -67,23 +107,21 @@ Promise.all(
   .then(docs => {
     return _.compact(docs.map(doc => {
       const measure = doc.QualityMeasureDocument;
-      const emeasureid = measure.subjectOf[0].measureAttribute[0].value[0].$.value;
+      const measureId = measure.subjectOf[0].measureAttribute[0].value[0].$.value;
       const strata = extractStrata(measure);
```

Comment:
What do you think about `const strataMaps = extractStrataMaps` or something similar? That way it's clear this isn't just individual strata, or a collection of just strata.